docs/_news/news12.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: news
+title: New for Codewind 0.12.0
+description: New for Codewind 0.12.0
+keywords: news, new, updates, update, version, IntelliJ
+duration: 1 minute
+permalink: news12
+---
+
+## Codewind 0.12.0
+Thursday 14, May 2020
+
+#### ✨ New Features and Highlights for 0.12.0 ✨
+**IntelliJ Tech Preview Updates**
+We're working to bring you a true Codewind experience with IntelliJ and are pleased to present:
+
+- Codewind for IntelliJ [supports the application metrics dashboard and the **Inject metrics** function](https://github.com/eclipse/codewind/issues/2386).
+- You can [open a shell into the running application container](https://github.com/eclipse/codewind/issues/2030)
+- You will be [notified of log changes, even when the log is not currently selected](https://github.com/eclipse/codewind/issues/2269).
+- When a project is opened in a new window, via actions such as reload due to new projoect creation or restart in debug mode (etc), [the project will be pre-selected in the Codewind view](https://github.com/eclipse/codewind/issues/2037).
```

Comment:
Could we simplify this sentence to something like, "If you open a project in a new window with certain actions, the project is preselected in the Codewind view."?

Reply:
I think it is better to specify the certain actions, because I think this behavior occurs only when a new window is opened (which is a result opening a new project, or when restarting). Thoughts and suggestions?

Reply:
@jopit ?

Reply:
I think I prefer Sarah's sentence.

Also, in the original sentence, 'project' is spelled 'projoect'